packages/amplify-dynamodb-simulator/__test__/index.test.js

```diff
@@ -43,15 +43,16 @@ describe('emulator operations', () => {
     }
   };
 
-  beforeEach(ensureNoDbPath);
-  afterEach(ensureNoDbPath);
-
   let emulators;
   beforeEach(() => {
+    ensureNoDbPath();
     emulators = [];
     jest.setTimeout(40 * 1000);
   });
-  afterEach(async () => await Promise.all(emulators.map(emu => emu.terminate())));
+
+  afterEach(async () => {
+    await Promise.all(emulators.map(emu => emu.terminate()));
```

Comment:
I think we still want `ensureNoDbPath()` to be called after the emulators are terminated.

Reply:
For some reason, adding ensureNoDbPath() at the end causes an error:
` EBUSY: resource busy or locked, unlink 'C:\Users\Stuti\Desktop\amplify-cli\packages\amplify-dynamodb-simulator\__test__\dynamodb-data\2080\fake_us-fake-1.db`